java/team25core/DeadmanMotorTask.java

```diff
@@ -155,15 +180,22 @@ protected void toggleMotor(GamepadTask.EventKind kind)
         case RIGHT_BUMPER_DOWN:
         case LEFT_TRIGGER_DOWN:
         case RIGHT_TRIGGER_DOWN:
-
+        case LEFT_STICK_DOWN:
+        case LEFT_STICK_UP:
+        case LEFT_STICK_RIGHT:
+        case LEFT_STICK_LEFT:
+        case RIGHT_STICK_DOWN:
+        case RIGHT_STICK_UP:
+        case RIGHT_STICK_RIGHT:
+        case RIGHT_STICK_LEFT:
             if (minPosition != MINMAX_POSITION_NOT_SET) {
-                currPosition = motor.getCurrentPosition();  //if the position is at a certain number which is the min height then it will stop
-                if (currPosition < minPosition) {
-                    positionWithinLimits = false;
+            currPosition = motor.getCurrentPosition();  //if the position is at a certain number which is the min height then it will stop
+            if (currPosition < minPosition) {
```

Comment:
This is awesome Shreya.  Good work.

I am however going to ask you to fix the indentation here.  You should indent starting on line 192 as that line is within the if clause that starts on 191.   It's hard to see where your compound statements begin and end without the braces lining up properly.

What's a compound statement you ask?  See Section 2.6.2 here https://docstore.mik.ua/orelly/java-ent/jnut/ch02_06.htm